src/app/page.tsx

```diff
@@ -4,7 +4,7 @@ export default function HomePage() {
   return (
     <main>
       <div className="container flex flex-col items-center justify-center gap-12 px-4 py-16">
-        <h5>Job listing page</h5>
```

Comment:
Maybe this code would be cleaner/more efficient if you did it like this:

`blahblah`

Reply:
oh yeah that makes sense